docs/editable_sections.md

```diff
@@ -0,0 +1,174 @@
+# Editable Sections ideas
```

Comment:
I added the idea to the docs but let me know if I should remove. I do miss ADR. Maybe we should talk about ADR and docs? Happy what the team decides to do with